src/rpcwallet.cpp

```diff
@@ -2142,3 +2142,35 @@ Value makekeypair(const Array& params, bool fHelp)
     result.push_back(Pair("PublicKey", HexStr(key.GetPubKey().Raw())));
     return result;
 }
+
+Value burn(const Array& params, bool fHelp)
+{
+    if (fHelp || params.size() < 1 || params.size() > 2)
+        throw runtime_error(
+            "burn <amount> [hex string]\n"
+            "<amount> is a real and is rounded to the nearest 0.00000001"
+            + HelpRequiringPassphrase());
+
+    if (pwalletMain->IsLocked())
+        throw JSONRPCError(RPC_WALLET_UNLOCK_NEEDED, "Error: Please enter the wallet passphrase with walletpassphrase first.");
+
+    int64_t nAmount = AmountFromValue(params[0]);
+    CScript scriptPubKey;
+
+    if (params.size() > 1) {
```

Comment:
could we modify this to conform to the coding that we been doing regarding {'s and }'s please and thanks. Makes it a little more cleaner.

also below the one statement after "else" can we remove the { } as its a single statement and is not needed to be blocked. Besides that this looks sound and look forward to testing this out.

good work.

Reply:
I will change that.